docs/RELEASE-NOTES.rst

```diff
@@ -6,15 +6,17 @@ next release
 Added Functionality
 `````````````````````
 * Changed container base image from debian-stretch to debian-buster
-* Support for using AS3 as an orchestration backend for Openshift Routes (Experimental)
+* Support for using AS3 as an orchestration backend for Openshift Routes
```

Comment:
* Added switch functionality between AS3 or CCCL with '--agent=as3/cccl' option in Controller deployment.

Reply:
We are going to specify that in Documentation. As part of release notes it is about the new backend that we added.

Reply:
agree. but also a new argument we added.